src/cmd/syslog-binding-cache/app/syslog_binding_cache.go

```diff
@@ -59,11 +59,11 @@ func (sbc *SyslogBindingCache) Run() {
 
 	go poller.Poll()
 
-	router := mux.NewRouter()
-	router.HandleFunc("/bindings", cache.LegacyHandler(legacyStore)).Methods(http.MethodGet)
-	router.HandleFunc("/v2/bindings", cache.Handler(store)).Methods(http.MethodGet)
-	router.HandleFunc("/aggregate", cache.LegacyAggregateHandler(aggregateStore)).Methods(http.MethodGet)
-	router.HandleFunc("/v2/aggregate", cache.AggregateHandler(aggregateStore)).Methods(http.MethodGet)
+	router := chi.NewRouter()
+	router.HandleFunc("/bindings", cache.LegacyHandler(legacyStore))
```

Comment:
How about using `router.Get` instead of `router.HandleFunc`? That would keep the behaviuor of only allowing `GET` requests.

Reply:
I figured it didn't matter, but done.